src/Maestro/Client/src/MaestroApi.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Sockets;
+using Newtonsoft.Json.Linq;
+using System.Threading.Tasks;
+
+
+namespace Microsoft.DotNet.Maestro.Client
+{
+    partial class MaestroApiResponseClassifier
+    {
+        public override bool IsRetriableException(Exception exception)
+        {
+            return base.IsRetriableException(exception) ||
+                exception is TaskCanceledException ||
```

Comment:
TaskCanceledException IS an OperationCanceledException, so line 21 is redundant. :-)